frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/prefill-vm-template-state-update.ts

```diff
@@ -116,6 +117,10 @@ export const prefillVmTemplateUpdater = ({ id, dispatch, getState }: UpdateOptio
       value: provisionSourceDetails.type ? provisionSourceDetails.type.getValue() : null,
     };
 
+    // update hostname
+    const hostname = getTemplateHostname(userTemplate);
```

Comment:
we should use a cloudinit hostname if the YAML one is missing to be compliant with the basic flow